### ISSUE TITLE
Handle PPC64 and ARM

### ISSUE THREAD

**Issue by dinosaure** [dinosaure/art#4]
#### PPC64

Nothing exists at this time about PPC64.

#### ARM

We should be able to support ARM if, at least, NEON is available but the CI fails on that.

**Comment by dinosaure** [comment]
`x86_32` is supported only if it supports SSE2 (but OCaml-CI provides one which does not handle SSE2).
`arm64` should be supported (at least, we have a mapping between SSE and NEON instructions) - but we must assert aligned read (which is not the case).

**Comment by jserv** [comment]
It seems that [lib/sse2neon.h](https://github.com/dinosaure/art/blob/master/lib/sse2neon.h) was taken from [SSE2NEON](https://github.com/DLTcollab/sse2neon). Can you keep the license notice and/or upstream information in hyperlink?

**Comment by dinosaure** [comment]
Yes of course, the state of the repo still is experimental, but I will add copyright and hyperlink to SSE2NEON 👍

**Comment by dinosaure** [comment]
Done by b9bb207, the issue still is open because the ARM support is not yet available.

**Comment by jserv** [comment]
> Done by [b9bb207](https://github.com/dinosaure/art/commit/b9bb20777edb35a5f3ae8da680efe608c0789c5c), the issue still is open because the ARM support is not yet available.

Thanks a lot. I look forward to the Arm support of this project.

**Comment by dinosaure** [comment]
ARM64 is supported 🎉 (see #33). We will start to look about PPC64.
